pkg/common/config/kafkaconfigmapwatcher.go

```diff
@@ -47,3 +48,15 @@ func InitializeKafkaConfigMapWatcher(ctx context.Context, watcher configmap.Watc
 
 	return nil
 }
+
+func InitializeObservabilityMapWatcher(ctx context.Context, watcher configmap.Watcher, logger *zap.SugaredLogger, handler LoggingObserver, namespace string) error {
```

Comment:
This could be refactored with the other other observer to have a generic func but didnt want to make more changes due to name changing across the codebase as this will be just a patch.